lib/screens/collection.dart

```diff
@@ -11,36 +15,32 @@ class CollectionScreen extends StatefulWidget {
   State<CollectionScreen> createState() => _CollectionScreenState();
 }
 
-class CapturedBeastie {
```

Comment:
I wasn't sure why this had it's own class instead of just using a generic Beastie model so I am removing it.